app/graphql/types/integrations/anrok.rb

```diff
@@ -30,7 +30,7 @@ def external_account_id
       end
 
       def failed_invoices_count
-        object.error_details.where(owner_type: 'Invoice').kept.count
+        Invoice.where(id: object.error_details.where(owner_type: 'Invoice').select(:owner_id)).failed.count
```

Comment:
can't we make it easier without sub-query? e.g. `object.organization.invoices.failed.count`?

Reply:
My concern was that potentially not all invoices can be failed because of tax error (in future). Having this we're sure that we're showing only the failed invoices due to errors with this integration, and anyway AR transforms it all into one query only